pages/osx/awk.md

```diff
@@ -0,0 +1,24 @@
+# awk
+
+> Find and Replace text within file(s).
+> More information: <https://ss64.com/osx/awk.html>.
+
+- Define the input field separator = regular expression fs:
+
+`awk [ -F fs ]`
```

Comment:
We don't use the `[ ]` syntax for optional arguments. Also, could you please write the descriptions in a more descriptive way? Currently, it sounds like a man page, which we try to avoid.